hw/formal/README.md

```diff
@@ -301,6 +302,31 @@ influence for "u_sha2.u_pad.shaf_ren".
 ...
 ```
 
+## How To Run VC Formal
+
+If you have access to VC Formal from Synopsys, you can formally verify your
+assertions. For example, to run formal property verification (FPV) using
+VC Formal on module `gpio`, type:
+```
+  cd hw/formal
+  fpv gpio
```

Comment:
This command is exactly the same for JasperGold as well? How does one distinguish between the two?

Reply:
We will change the `fpv` and `fpv_all` files to add a knob for vc_formal. With that PR, we will change the description here. Maybe it is good to add a TODO here.